daq_macros.py

```diff
@@ -2201,14 +2248,19 @@ def runRasterScan(currentRequest,rasterType=""): #this actually defines and runs
     rasterReqID = defineRectRaster(currentRequest,10,290,10)    
     RE(snakeRaster(rasterReqID))
     daq_lib.set_field("xrecRasterFlag","100")    
+  elif (rasterType=="Custom"):
+    daq_lib.set_field("xrecRasterFlag","100")
+    beamline_lib.mvrDescriptor("omega",omega_rel)
+    rasterReqID = defineRectRaster(currentRequest, width+step_size, height+step_size, step_size)
+    RE(snakeRaster(rasterReqID))
   else:
     rasterReqID = getXrecLoopShape(currentRequest)
     logger.info("snake raster " + str(rasterReqID))
     time.sleep(1) #I think I really need this, not sure why
     RE(snakeRaster(rasterReqID))
 
 def gotoMaxRaster(rasterResult,multiColThreshold=-1,**kwargs):
-  global autoVectorCoarseCoords,autoVectorFlag
+  global autoVectorCoarseCoords,autoVectorFlag, max_col
```

Comment:
gotoMaxRaster is a real mess. this needs to be refactored (as we had seen with the multiCol upgrade attempts before)

Reply:
I will attempt to refactor it in the next PR